source/stylesheets/modules/_country_filter.scss

```diff
@@ -4,56 +4,43 @@
 @import "settings/grid_layout";
 
 .country-filter {
+
+  position: relative;
+
   > .headline {
     @include screenreaders-only;
   }
 
   > .select {
-    background: $box-background;
-    min-height: $gutter;
-
-    .ng-ms {
-      border: none;
-      box-shadow: none;
-      position: relative;
-
-      // Adds CSS triangle to the select box
-      &:before {
-        position: absolute;
-        top: calc(50% - 1px); // always center it
-        right: 8px;
-        display: inline-block;
-        border-right: 5px solid transparent;
-        border-top: 5px solid #CCC;
-        border-left: 5px solid transparent;
-        border-bottom-color: rgba(0, 0, 0, 0.2);
-        content: '';
-      }
-
-      input {
-        background: inherit;
-        height: $gutter;
-        margin-top: 0;
-        line-height: $gutter;
-
-        &:hover {
-          @include placeholder {
-            color: $text-color;
-          }
-        }
-      }
+    display: block;
+    width: 100%;
+    height: $gutter;
+    padding-left: 6px;
+    color: $text-color;
+    border: none;
+    border-radius: 0px; // Removes Chrome border-radius
```

Comment:
Yeah, I used to do that but read somewhere about that its a bad idea. Don't know anymore why :-)